proto/buf/alpha/registry/v1alpha1/schema.proto

```diff
@@ -0,0 +1,196 @@
+// Copyright 2020-2022 Buf Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package buf.alpha.registry.v1alpha1;
+
+import "google/protobuf/descriptor.proto";
+
+// Provides endpoints for downloading dynamic schemas and for using schemas
+// in validation and data transformation functions.
+service SchemaService {
+  // GetSchema allows the caller to download a schema for one or more requested
+  // types, RPC services, or RPC methods.
+  rpc GetSchema(GetSchemaRequest) returns (GetSchemaResponse);
+
+  // Reformat allows the caller to indicate a type schema which will be used to
+  // transform given data to a different format.
+  rpc Reformat(ReformatRequest) returns (ReformatResponse);
```

Comment:
Is `reformat` the best we can do? also, reformat what? it's a `SchemaService` so I'd be forgiven to assume its reformatting my schema. Following a verb+noun pattern...

**Verb**
- Translate
- Reformat
- Render
- Convert
- Decode
- Decipher

**Noun**
- Data
- Payload
- Message

I like `translateMessage` if we are trying to differ from the existing `convert` endpoint but otherwise `convert` is an equally appropriate verb.

Reply:
We have Convert in https://github.com/bufbuild/buf/blob/936c01c610632585a71c2b3a4cf4cea6a0db42b9/proto/buf/alpha/registry/v1alpha1/convert.proto#L28-L33, which I think is what @jhump is trying to avoid a conflict with. I don't think that api is really used since it was put out, and we may be able deprecate/remove that one.

Reply:
I have made TCN-735 and am looking into this now.

Reply:
Changed this to `ConvertMessage`.